ops/framework.py

```diff
@@ -199,6 +199,11 @@ def __init__(self, parent, key):
         else:
             self.framework = parent.framework
             self.handle = Handle(parent, kind, key)
+        if parent is not self:
+            # Framework.__init__ passes parent=self, and we don't want to track ourself (also framework._objects may not be initialized yet)
```

Comment:
Given the above, this should probably be reworked to something like:

```python
# We don't want to track the Framework itself, since it is the root of the tree and framework._objects may even not be initialized yet
```